indexer/editable_map_object.hpp

```diff
@@ -101,7 +101,7 @@ class EditableMapObject : public MapObject
   std::vector<MetadataID> GetEditableProperties() const;
 
   /// See comment for NamesDataSource class.
-  NamesDataSource GetNamesDataSource(bool addFakes = true);
+  NamesDataSource GetNamesDataSource(bool addFakes = false);
```

Comment:
I will do that later, because it would brake the ios version if I did it now

Reply:
@kirylkaveryn can you please help to remove this parameter? Or was it already removed?

Reply:
Because it is not used anymore and completely removed from Android code. All related unused code should also be removed for iOS (when `addFakes == true`).

Reply:
The value of `needFakes` is ignored in the core, and you can also call the function with out a parameter. So you can safely remove all `needFakes` related code from iOS

Reply:
@map-per  Fixed!
You can pick up the fix from my last commit here: https://github.com/organicmaps/organicmaps/pull/7946/commits/aee10205fe6db5f7d29ca9a9c8a8d82922f5f48c

So you can safely remove the `needFakes` form cpp now.